storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -34,22 +34,11 @@ You can provision your {product-title} cluster with persistent storage by using
 endif::openshift-rosa,openshift-dedicated[]
 
 The Kubernetes persistent volume framework allows administrators to provision a cluster with persistent storage and gives users a way to request those resources without having any knowledge of the underlying infrastructure.
-You can dynamically provision AWS EBS volumes.
+You can dynamically provision Amazon EBS volumes.
 Persistent volumes are not bound to a single project or namespace; they can be shared across the {product-title} cluster.
 Persistent volume claims are specific to a project or namespace and can be requested by users.
 You can define a KMS key to encrypt container-persistent volumes on AWS.
-
-[IMPORTANT]
-====
-{product-title} defaults to using an in-tree, or non-Container Storage Interface (CSI) plug-in to provision AWS EBS storage. In future {product-title} versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. 
-
-CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes.
-ifndef::openshift-rosa,openshift-dedicated[]
-For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
-endif::openshift-rosa,openshift-dedicated[]
-
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
-====
+By default, newly created clusters using ROSA version 4.10 and later use gp3 storage and the Amazon Elastic Block Store (Amazon EBS) Container Storage Interface (CSI) driver.
```

Comment:
```suggestion
By default, newly created clusters using {product-title} version 4.10 and later use gp3 storage and the Amazon Elastic Block Store (Amazon EBS) Container Storage Interface (CSI) driver.
```

Reply:
@mletalie I think? Ponder on it.

Reply:
Hello @GroceryBoyJr, eh.... good point. Let me look into it. Appreciate it!